src/traces/table/plot.js

```diff
@@ -0,0 +1,892 @@
+                d.calcdata.columnDragInProgress = false;
+                easeColumn(movedColumn, d, 0);
+                columnMoved(gd, calcdata, p.key, p.columns.map(function(dd) {return dd.xIndex;}));
+            })
+        );
+
+    if(!c.clipView) {
+        yColumn.attr('clip-path', function(d) {return 'url(#columnBoundaryClippath_' + d.specIndex + ')';});
+    }
+
+    var columnBlock = yColumn.selectAll('.columnBlock')
+        .data(splitToPanels, gup.keyFun);
+
+    columnBlock.enter()
+        .append('g')
+        .classed('columnBlock', true)
+        .attr('id', function(d) {return d.key;})
+        .style('user-select', 'none');
+
+    columnBlock
+        .style('cursor', function(d) {
+            return d.dragHandle ? 'ew-resize' : d.calcdata.scrollbarState.barWiggleRoom ? 'ns-resize' : 'default';
+        });
+
+    var cellsColumnBlock = columnBlock.filter(cellsBlock);
+
+    cellsColumnBlock
+        .call(d3.behavior.drag()
+            .origin(function(d) {
+                d3.event.stopPropagation();
+                return d;
+            })
+            .on('drag', makeDragRow(gd, tableControlView, -1))
+            .on('dragend', function() {
+                // fixme emit plotly notification
+            })
+        );
+
+    // initial rendering: header is rendered first, as it may may have async LaTeX (show header first)
+    // but blocks are _entered_ the way they are due to painter's algo (header on top)
+    renderColumnCellTree(gd, tableControlView, columnBlock.filter(headerBlock), columnBlock);
+    renderColumnCellTree(gd, tableControlView, columnBlock.filter(cellsBlock), columnBlock);
+
+    var scrollAreaClip = tableControlView.selectAll('.scrollAreaClip')
+        .data(gup.repeat, gup.keyFun);
+
+    scrollAreaClip.enter()
+        .append(c.clipView ? 'g' : 'clipPath')
+        .classed('scrollAreaClip', true)
+        .attr('id', function(d) { return 'scrollAreaBottomClip_' + d.key;});
+
+    var scrollAreaClipRect = scrollAreaClip.selectAll('.scrollAreaClipRect')
+        .data(gup.repeat, gup.keyFun);
+
+    scrollAreaClipRect.enter()
+        .append('rect')
+        .classed('scrollAreaClipRect', true)
+        .attr('x', -c.overdrag)
+        .attr('y', -c.uplift)
+        .attr('stroke', 'orange')
+        .attr('stroke-width', 2)
+        .attr('fill', 'none')
+        .style('pointer-events', 'stroke');
+
+    scrollAreaClipRect
+        .attr('width', function(d) {return d.width + 2 * c.overdrag;})
+        .attr('height', function(d) {return d.height + c.uplift;});
+
+    var columnBoundary = yColumn.selectAll('.columnBoundary')
+        .data(gup.repeat, gup.keyFun);
+
+    columnBoundary.enter()
+        .append('g')
+        .classed('columnBoundary', true);
+
+    var columnBoundaryClippath = yColumn.selectAll('.columnBoundaryClippath')
+        .data(gup.repeat, gup.keyFun);
+
+    // SVG spec doesn't mandate wrapping into a <defs> and doesn't seem to cause a speed difference
```

Comment:
Please add your `<clipPath>` nodes in `fullLayout._clips`, for consistency with other trace types.